src/index.js

```diff
@@ -28,6 +28,15 @@ module.exports = function(session) {
         throw new Error('No dataset provided to Datastore Session.');
       }
       this.kind = options.kind || 'Session';
+      if (
+        options.expirationMs &&
+        (typeof options.expirationMs !== typeof 0 || options.expirationMs < 0)
```

Comment:
I would probably allow someone to set `0`, if they want to be explicit about the the session not expiring.

I would then just have a check that throws if:

```
if (typeof options.expirationMs !== 'number' || Number(options.expirationMs) <= 0)
```

Reply:
This version does accept a `0` as an input - numeric zero is falsey so the error won't be thrown. Javascript is not the most intuitive language, I'd be happy to make that easier in whatever way you'd like.

Reply:
I've rewritten this chunk of code to hopefully be easier to read.

```javascript
      if (options.expirationMs) {
        if (
          typeof options.expirationMs !== typeof 0 ||
          options.expirationMs < 0
        ) {
          throw new Error(
            'invalid value for option expirationMs: must be an integer greater than 0'
          );
        }
        this.expirationMs = parseInt(options.expirationMs, 10);
      } else {
        this.expirationMs = 0;
      }
```

Side note: I prefer `typeof myvar === typeof 0` or `typeof myvar === typeof Number()` to `typeof myvar === 'number'` because:

- I don't have to remember the names or casing of the type names
- I don't have to worry about some future version of javascript breaking it

It's probably only an academic concern, again I'm happy to defer to the package maintainers here.